src/main/java/edu/jhu/clueless/ClientAction.java

```diff
@@ -0,0 +1,44 @@
+package edu.jhu.clueless;
```

Comment:
how will this transfer things like the location a player wants to move to or a card in their suggestion? is all this meant to be captured in the action string?